packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -854,12 +855,12 @@ contract VotingV2 is
             if (!_priceRequestResolved(priceRequest, voteInstance, currentRoundId)) {
                 // If the request is not resolved and the lastVotingRound less than the current round then the vote
                 // must have been rolled. In this case, update the internal trackers for this vote.
-
                 if (priceRequest.lastVotingRound < currentRoundId) {
                     priceRequest.lastVotingRound = SafeCast.toUint32(currentRoundId);
                     deletedRequests[requestIndex] = requestIndex;
                     priceRequest.priceRequestIndex = SafeCast.toUint64(priceRequestIds.length);
                     priceRequestIds.push(priceRequestIds[requestIndex]);
+                    indexTo++;
```

Comment:
One question @chrismaree : why would we need this line?
Would there be a problem if the last request is rolled?

Reply:
because we incremented the length of `priceRequestIds` we need to increment this as well as in the normal case the `indexTo` is equal to `priceRequestIds` length. I'll add a comment explaining this.

Reply:
This has been further refined! we only need to increment this iff this method was entered via the `updateTrackers` call as this flow expects to traverse _all_ price requests.

Reply:
Makes sense! So, it won't change the original goal of `updateTrackersRange`, which was to handle a certain number of price requests.